resources/scripts/artifacts-api-latest-release-versions.sh

```diff
@@ -28,7 +28,8 @@
 set -eo pipefail
 
 URL="https://artifacts-api.elastic.co/v1"
+NO_KPI_URL_PARAMS="x-elastic-no-kpi=true"
 
 OUTPUT=latest-release-versions.json
 
-curl -s ${URL}/versions | jq -r '.versions[] | select(contains("SNAPSHOT")|not)' | jq -R . | jq -s '. | sort' | tee ${OUTPUT}
+curl -s "${URL}/versions?${NO_KPI_URL_PARAMS}" | jq -r '.versions[] | select(contains("SNAPSHOT")|not)' | jq -R . | jq -s '. | sort' | tee ${OUTPUT}
```

Comment:
I'm adding the parameter separator here instead of in the variable to make a clear separation on where the params do start